modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssueDetail.js

```diff
@@ -60,3 +69,137 @@ HtmlPanel.propTypes = {
 	content: PropTypes.string.isRequired,
 	title: PropTypes.string.isRequired,
 };
+
+const FailingElementsPanel = ({failingElements, issueType}) => {
+	const [shownElements, setShownElements] = useState(10);
+
+	const normalizedFailingElements = normalizeFailingElements(
+		failingElements,
+		issueType
+	);
+
+	const totalElements = normalizedFailingElements.length;
+
+	const onViewMore = () => {
+		const newShownElements = shownElements + 10;
+
+		setShownElements(
+			newShownElements < totalElements ? newShownElements : totalElements
+		);
+	};
+
+	return (
+		<ClayPanel
+			collapsable
+			collapseClassNames="c-mb-4 c-mt-3"
```

Comment:
Maybe it is some kind of conversion but why here we prefix the bootstrap margin classes but in other places (like FailingElement component) not?

Reply:
You are right, I neither know what the prefix is for. I'm removing it.